lib/utils/ast-utils.ts

```diff
@@ -0,0 +1,718 @@
+/* tslint:disable */
```

Comment:
this file is taken from angular and being used in multiple files that's why /* tslint:disable */ is added.
I have modified the module to nest for our project.